sources/packages/backend/libs/test-utils/src/factories/sfas-application.ts

```diff
@@ -0,0 +1,40 @@
+import { SFASApplication } from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import * as faker from "faker";
+
+/**
+ * Create fake SFAS application.
+ * @param options SFAS application options.
+ * - `initialValues` SFAS application initial values.
+ * @returns persisted SFAS application.
+ */
+export function createFakeSFASApplication(options?: {
+  initialValues?: Partial<SFASApplication>;
+}): SFASApplication {
+  const sfasApplication = new SFASApplication();
+  sfasApplication.id = faker.datatype.number({
```

Comment:
Please avoid creating id in the fake factory. I know there is one or more implementation setting id. But please do not set unless there is a particular need.

Reply:
Creating id is mandatory here because the id is imported from the SFAS files. It is the same for SFAS individual: [here](https://github.com/bcgov/SIMS/blob/6d9a295f2245cf0536ba4a68695b9fbc8b3a3f42/sources/packages/backend/libs/test-utils/src/factories/sfas-individuals.ts#L16).

Reply:
Isn't the id auto-generated if not supplied?? are you getting any error?

Reply:
My bad. I see that the id column is not auto-generated.

![image](https://github.com/bcgov/SIMS/assets/54600590/8dc97693-05b2-4877-9ef2-85e462660969)